core/common/library/Google/Analytic.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Phanbook : Delightfully simple forum software
+ *
+ * Licensed under The GNU License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @link    http://phanbook.com Phanbook Project
+ * @since   1.0.0
+ * @license http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
+ */
+namespace Phanbook\Google;
+use Phanbook\Models\Settings;
+
+/**
+*
+*/
+class Analytic extends \Phalcon\DI\Injectable
```

Comment:
+class Analytic extends \Phalcon\DI\Injectable to class Analytic extends Injectable, then add use \Phalcon\DI\Injectable the above